doc/devbox_setup.md

```diff
@@ -12,7 +12,7 @@ This document describes how to prepare your development environment to use the *
 
 ## Set up a Windows development environment
 
-- Install [Visual Studio 2017][visual-studio]. You can use the **Visual Studio Community** Free download if you meet the licensing requirements.  (**Visual Studio 2015** is also supported.)
+- Install [Visual Studio 2019][visual-studio]. You can use the **Visual Studio Community** Free download if you meet the licensing requirements.  (**Visual Studio 2015** and **Visual Studio 2017**  are also supported.)
```

Comment:
Consolidate 2 spaces to 1 between "**Visual Studio 2017** and "are also supported"